requirements-dev.in

```diff
@@ -4,6 +4,3 @@ pycurl
 pytest
 tox
 tox-gh-actions
-typing; python_version < "3.0"
```

Comment:
You need to run `pip-compile --no-annotate requirements-dev.in` so these changes are applied.

Reply:
You are missing the changes in `requirements.in` as well.